### PR TITLE
Set error status when running non-scratch production build

### LINKED ISSUE #848: Document that we can't run production (non-scratch) builds from upstream
It looks (#842) like we can't do non-scratch builds and we need to check with releng if and how they can allow that. (See https://pagure.io/releng/issue/9801 for more info.)

Until we need more, we need to:
- [ ] document the requirement for scratch=true in the docs
- [ ] put a link with a link to the pagure issue above when scratch=false is used.

### PR DESCRIPTION
**Pull request by lbarcziova**
Fixes #848

### CODE REVIEW

File: packit_service/worker/build/koji_build.py [hunk 1]
```diff
@@ -113,6 +113,17 @@ def supported_koji_targets(self):
         return self._supported_koji_targets
 
     def run_koji_build(self) -> TaskResults:
+        if not self.is_scratch:
+            msg = (
+                "It is not possible to run non-scratch production builds from upstream."
```
Comment (lachmanfrantisek) [inline packit_service/worker/build/koji_build.py:118]: I would try to make it shorter. The space is really limited on Github so I would try to put most important info to the beginning...

```suggestion
                "Non-scratch builds not possible from upstream."
```

**Review by lachmanfrantisek — COMMENTED** [review]
Nice!!

Just one suggestion about status text, otherwise LGTM

File: packit_service/worker/build/koji_build.py (hunk 1 above)
Comment (lbarcziova) [reply packit_service/worker/build/koji_build.py]: right, done

**Review by lachmanfrantisek — APPROVED** [review]
:+1:

**Pull request merged**